HelpDesk.App.PostgresSql/appsettings.macbook.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ConnectionStrings": {
```

Comment:
actually, you do not need to copy whole appsettings file, just things that are different from appsettings.json, such as RsaKeysPath